internal/rbd/rbd_util.go

```diff
@@ -306,74 +308,109 @@ func rbdStatus(ctx context.Context, pOpts *rbdVolume, cr *util.Credentials) (boo
 	return false, output, nil
 }
 
-// rbdManagerTaskDelete adds a ceph manager task to delete an rbd image, thus deleting
-// it asynchronously. If command is not found returns a bool set to false
-func rbdManagerTaskDeleteImage(ctx context.Context, pOpts *rbdVolume, cr *util.Credentials) (bool, error) {
+// addRbdManagerTask adds a ceph manager task to execute command
+// asynchronously. If command is not found returns a bool set to false
+// example arg ["trash", "remove","pool/image"]
+func addRbdManagerTask(ctx context.Context, pOpts *rbdVolume, arg []string) (bool, error) {
```

Comment:
I would really like to see a single `error` returned, not `(bool, error)`. It can be `ErrNotSupported` instead of the potential `false`.

Reply:
This is not new implementation the code already , if any changes are required we can do it as an enhancement

Reply:
ok!